sycl/include/sycl/detail/pi.h

```diff
@@ -1308,6 +1310,19 @@ __SYCL_EXPORT pi_result piextMemCreateWithNativeHandle(
     pi_native_handle nativeHandle, pi_context context, bool ownNativeHandle,
     pi_mem *mem);
 
+/// Creates PI image object from a native handle.
+/// NOTE: The created PI object takes ownership of the native handle.
+///
+/// \param nativeHandle is the native handle to create PI image from.
+/// \param context The PI context of the memory allocation.
+/// \param ownNativeHandle Indicates if we own the native memory handle or it
+/// came from interop that asked to not transfer the ownership to SYCL RT.
+/// \param img is the PI img created from the native handle.
+__SYCL_EXPORT pi_result piextMemImgCreateWithNativeHandle(
```

Comment:
@kbenzie: please track yet another change coming to UR

Reply:
We are tracking this with https://github.com/oneapi-src/unified-runtime/issues/428